docs/getting-started.md

```diff
@@ -23,10 +23,34 @@ Building your own inventory
 
 Ansible inventory can be stored in 3 formats: YAML, JSON, or inifile. There is
 an example inventory located
-[here](https://github.com/kubernetes-incubator/kargo/blob/master/inventory/inventory.example).
+[here](https://github.com/kubernetes-incubator/kargo/blob/master/inventory/inventory.example):
 
-You can use an
+```
+cp inventory/inventory.example inventory.cfg
+```
+
+Or you can use an
 [inventory generator](https://github.com/kubernetes-incubator/kargo/blob/master/contrib/inventory_builder/inventory.py)
 to create or modify an Ansible inventory. Currently, it is limited in
 functionality and is only use for making a basic Kargo cluster, but it does
-support creating large clusters.
+support creating large clusters. For example:
+
+```
+python3 contrib/inventory_builder/inventory.py 10.10.1.3 10.10.1.4 10.10.1.5
```

Comment:
cd inventory && python3 ../contrib/inventory_builder/inventory.py 10.10.1.3 10.10.1.4 10.10.1.5

That will write the inventory.cfg in the right directory actually